paper/ms.tex

```diff
@@ -1874,23 +1886,23 @@ \section{Conclusions}
 \label{sec:conclusion}
 
 In this work, we searched 120-second cadence TESS data collected
-between July~2018 and Sep~2022 for complex quasiperiodic variables
+from 2018 July to 2022 September for complex quasiperiodic variables
 (CQVs).  The target stars were \nstarssearched\ K and M dwarfs within
 150\,pc.  The selection function was $>$$80\%$ complete within 30\,pc,
 and $<$$10\%$ complete at distances exceeding 100\,pc.
 
 In our target sample, we found \ngoods\ objects that showed complex
 quasiperiodic behavior over at least one TESS sector.  These \ngoods\
-bonafide CQVs are listed in Table~\ref{tab:thetable}.  This table also
+bona fide CQVs are listed in Table~\ref{tab:thetable}.  This table also
```

Comment:
https://www.google.com/search?q=bonafide+vs+bona+fide&oq=bonafide+vs+bona+fide&gs_lcrp=EgZjaHJvbWUqBwgAEAAYgAQyBwgAEAAYgAQyCAgBEAAYFhgeMggIAhAAGBYYHjIKCAMQABiGAxiKBTIKCAQQABiGAxiKBTIKCAUQABiGAxiKBTIKCAYQABiGAxiKBTIKCAcQABiGAxiKBdIBCDMzOThqMWoxqAIAsAIA&sourceid=chrome&ie=UTF-8

Reply:
https://english.stackexchange.com/questions/318650/bona-fide-vs-bonafide#:~:text=No.,'bonafide'%20if%20you%20wish. and https://www.oed.com/dictionary/bona-fide_adv?tab=factsheet#16784507

Reply:
I concede!  😂
(also checked some old papers in the AAS journals, and their copy editors agree with "bona fide"!)

Reply:
https://github.com/lgbouma/cpv/commit/dc8845f5e22dff0246e11b71df8be9bce3552dea